docs/SOPs/update_ena_runs_SOP.md

```diff
@@ -0,0 +1,99 @@
+   ```
+   git clone https://github.com/ebi-ait/ingest-archiver.git
+   pip install -r requirements.txt
+   ```
+5. Get your JWT Token from Ingest UI.
+   1. Log in in Ingest UI https://contribute.data.humancellatlas.org/login using the account which has the WRANGLER role.
+   1. In Chrome, right click and select _Inspect_ to open the developer console. Select the Network tab.
+   1. Refresh the page, `https://contribute.data.humancellatlas.org/home`
+   1. Check the _Authorization_ in headers of the request to `https://api.ingest.archive.data.humancellatlas.org/auth/account`
+   1. Copy the token after _Bearer_ prefix: 
+      ```
+      Authorization: Bearer <copy the very long string of random characters>
+      ```
+   1. That token has 1 hr validity. The token will be needed in the submitter script later.
+## Steps
+
+1. Get the list of sequencing runs to be suppressed. This can be downloaded as TSV/JSON from the ENA Browser. 
+   
+2. Make sure
+   that the metadata in Ingest contains sequencing experiment accessions. The submitter script will raise an error if
+   any of the assay processes has no accession. The assay processes in the submission should have the following
+   property:
+
+    ```
+    "insdc_experiment": {
+      "insdc_experiment_accession": "ERX4319109"
+    }
+    ```
+
+2. Clear the sequencing run accessions in file metadata. The following should not be in the file metadata json:
+    ```
+    "insdc_run_accessions": [
+      "ERR6449905"
+    ]
+    ```
+   
+   Update script to have a jwt token from the Ingest UI then run the following: 
+    ```
+    python clear_run_accession_from_files.py <submission-uuid>
+    ```
+
+4. Download all files from Ingest / Terra upload area to any directory inside `/nfs/production/hca/` in the EBI cluster.
+   The files may also be in the hca-util upload area but we should make sure they're valid. Using Ingest/Terra upload area means the files have already been validated before.
+   Please prefer downloading the Terra upload area as downloading from Ingest upload area will incur cost to our AWS account.
+
+5. Checksum all the files.
+    ``` 
+    $ cd to directory where you've downloaded the files 
+    $ md5sum * > <md5-filename>.txt
+    ```
+6. Upload the files to Webin FTP upload area (could be in parallel with checksumming)
+   ```
+   $ cd <directory where you downloaded the files>
+   $ lftp webin2.ebi.ac.uk -u <webin-user>
+   $ > # input webin-password
+   $ mkdir parent-dir
+   $ cd parent-dir
+   $ mput *
+   ```   
+   Please refer to [ENA documentation](https://ena-docs.readthedocs.io/en/latest/update/metadata/programmatic-read.html) for more details
+7. Run the submitter script. The `receipt.xml` and `report.json` file should be available after running the script.
+   The `receipt.xml` will contain the ENA REST API response. The `report.json` will contain some report on which files were updated with the run accessions from ENA response.
+   ```json
```

Comment:
why json? why not bash?

Reply:
sorry, I don't understand what you mean by this